MmSupervisorPkg/SupervisorPolicyTools/SupervisorPolicyMaker.py

```diff
@@ -164,13 +165,15 @@ def MakeSupervisorPolicy(output_version=Supervisor_Policy.FLEXBILE_STRUCTURE_VER
             ParseXmlAndAddToPolicy(xml_file_path, Policy)
 
             # print out our policy
-        print("=================================================")
-        print("========    Start Dumping Policy    =============")
-        print("=================================================")
-        Policy.DumpInfo(prefix="  ")
-        print("=================================================")
-        print("========    End Dumping Policy    ===============")
-        print("=================================================")
+        logging.debug("=================================================")
+        logging.debug("========    Start Dumping Policy    =============")
+        logging.debug("=================================================")
+        outfs = io.StringIO()

```

Comment:
Hmm, I would just keep it simple that the dumpinfo will also use logging.** functions.

Reply:
There are two reasons I went this route:

1. This method allows a caller to change a single line of code (`outfs=<something`) to dump to anything - stdout, stderror, a file, a log, etc). Changing it to hard code `logging.*` reduces it's reusability. Someone would have to completely rewrite / change 50+ lines of code to write to a different logging level or to a file
2. This method allows the `Print()` function to continue working as expected. If we hardcode it the way you are suggesting, we break the Print() function that exists for each of these objects. It would now just log it instead of print it.

If you are OK with both of these situations, I can go through and change it to logging.DEBUG or logging.VERBOSE